src/param.rs

```diff
@@ -271,12 +271,12 @@ pub struct AudioParam {
 // helper struct to attach / detach to context (for borrow reasons)
 #[derive(Clone)]
 pub(crate) struct AudioParamRaw {
```

Comment:
Rename to `AudioParamInner`? Would be more consistent with the reset of the codebase